pyglmnet/pyglmnet.py

```diff
@@ -505,14 +514,28 @@ def pseudo_R2(self, y, yhat, ynull):
 
         elif self.distr == 'normal':
             R2 = 1 - np.sum((y - yhat)**2) / np.sum((y - ynull)**2)
+        elif self.distr == 'multinomial':
+            y = label_binarizer(y)
+            # yhat is the probability of each output
+            if yhat.ndim != y.ndim or ynull.ndim != y.ndim:
+                raise Exception(('yhat and ynull must '
+                                 'be a (n_samples, n_class) '
+                                 'matrix with the prediction for '
+                                 'each class'
+                                 ))
```

Comment:
you can define a variable called `msg` which is the string inside `Exception` and then give it as input to `Exception`. That way, you won't have wrap over too many lines in order to conform to pep8 ...